### ISSUE TITLE
Is the material "samples" property still used?

### ISSUE THREAD

**Issue by Theverat** [LuxCoreRender/LuxCore#154]
Materials (and lights) have a "samples" property:
https://github.com/LuxCoreRender/LuxCore/blob/master/include/slg/materials/material.h#L199

Is this still used by any engines?
I have the feeling that it was only used by TilePath and is no longer supported, but I could be wrong.

**Comment by Dade916** [comment]
Yup, they are not used anymore. It was something supported by the old mega-kernel TILPATHOCL. I guess I can remove them.
